components/grid/grid.css

```diff
@@ -0,0 +1,7 @@
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(20rem, 1fr));
+  column-gap: 1rem;
+  row-gap: 2rem;
```

Comment:
Can we use `gap: 2rem 1rem` as a shorthand here.

Reply:
thanks jeet, I have made those changes, please review